app/src/main/java/com/x8bit/bitwarden/data/platform/manager/model/FlagKey.kt

```diff
@@ -19,6 +19,15 @@ sealed class FlagKey<out T : Any> {
      */
     abstract val isRemotelyConfigured: Boolean
 
+    /**
+     *  Data object holding the key for syncing with the Bitwarden Authenticator app.
+     */
+    data object AuthenticatorSync : FlagKey<Boolean>() {
```

Comment:
I think we may need to re-base, not ideal but there is a couple steps to do if we wanted to add this as toggle-able in the debug menu

Reply:
Ah nice callout! Somebody mentioned that to me last week, but I forgot about it. I'll rebase now!

Reply:
Rebase and updated a few things so that the flag shows up in the debug menu (and the changes get observed, too!)

Nice to see the work you did setting that up is already paying off 🤗